kytos/core/controller.py

```diff
@@ -157,9 +157,38 @@ def __init__(self, options=None, loop=None):
 
     def enable_logs(self):
         """Register kytos log and enable the logs."""
+        decorators = self.options.logger_decorators
+        decorators = [self._resolve(decorator) for decorator in decorators]
```

Comment:
If users accidentally set a wrong value for `logger_decorators` it can make `kytosd` to keep hanging:

```
Web update - Web UI was not updated
Exception in callback Controller.start()
handle: <Handle Controller.start()>
Traceback (most recent call last):
  File "/usr/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/viniarck/repos/kytos/kytos/core/controller.py", line 242, in start
    self.enable_logs()
  File "/home/viniarck/repos/kytos/kytos/core/controller.py", line 161, in enable_logs
    decorators = [self._resolve(decorator) for decorator in decorators]
  File "/home/viniarck/repos/kytos/kytos/core/controller.py", line 161, in <listcomp>
    decorators = [self._resolve(decorator) for decorator in decorators]
  File "/home/viniarck/repos/kytos/kytos/core/controller.py", line 179, in _resolve
    __import__(used)
ModuleNotFoundError: No module named 'kytos.core.logger_decorators.queue_decoratorx'; 'kytos.core.logger_decorators' is not a package

```

Could you handle the exception and then call `sys.exit` passing the error, something similar to https://github.com/kytos-ng/kytos/blob/master/kytos/core/controller.py#L337-L342? Other options on kytos.conf aren't also handling errors we might improve the other ones in he future, but since this one here can hang we want to avoid that type of surprise, especiflly if it's running a container it can take some time for operators to realize that an issue happened during initialization.

Reply:
Added handling for both the case of `ModuleNotFoundError` and `AttributeError`.